### DISCUSSION TITLE
Ring Extender 2nd Gen - alternate ID - best practice

### DISCUSSION THREAD

**Discussion by hiddeninplainsight33** [zwave-js/zwave-js#5363] [Q&A] [answered]
Hi all - total noob here so please forgive any stupidity.

I have a Ring Range Extender V2 - but it is reporting an alternative product ID to the existing database entry at 

https://github.com/zwave-js/node-zwave-js/blob/f7f50a4eac0af000033a0b68d1443e2c969059ee/packages/config/config/devices/0x0346/range_extender_gen2.json

Existing product ID on the entry is 0x0301, the recently purchased device is reporting 0x0401 (full string 0x0346:0x0401:0x0401).

I'm assuming given the multiple ID's on the previous gen v1 (https://github.com/zwave-js/node-zwave-js/blob/f7f50a4eac0af000033a0b68d1443e2c969059ee/packages/config/config/devices/0x0346/range_extender_gen1.json) that it is good practice to create a second entry for the newer product ID - but given the ZWave alliance ID is needed and this new variant of product ID doesn't show on that DB (https://products.z-wavealliance.org/products/4148?selectedFrequencyId=1) is it still acceptable to do so?  Or do I have to wait for it to be on the ZWA DB?

Hope this makes some sense - and apologies for the noob-ness.....

**Comment by AlCalzone** [comment, marked as answer]
The zwave alliance id isn't required, it's nice to have for cross referencing.
You can just add the additional device id to the file.
